cli-feed-v4.json

```diff
@@ -15621,7 +15621,7 @@
               "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
             },
             "windowsSiteConfig": {
-              "netFrameworkVersion": "v6.0"
+              "netFrameworkVersion": "v7.0"
```

Comment:
We need a whole new entry to accompany this, right?

Reply:
Do we need net7-isolated with both? - netFrameworkVersion 6.0 and 7.0?

Reply:
I just mean that the fix should be applied in a new overall 4.47.0 entry rather than in an existing entry that customers may have cached locally already.